Practical Example/RWTakeHome/RWTakeHome/NetworkService/NetworkService.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  Networking.swift
+//  RWTakeHome
+//
+//  Created by Nick Nguyen on 10/7/20.
+//
+
+import Foundation
+
+class NetworkService {
+
+  private let jsonDecoder = JSONDecoder()
+
+  private var SECRET_KEY: String {
+    let env = ProcessInfo.processInfo.environment
+    return env["SecretKey"] ?? "NO JSON_BIN.IO SECRET KEY FOUND"
+  }
+
+  private func generateURLForTutorial(ofType type: TutorialType) -> URLRequest {
+    //https://api.jsonbin.io/b/5f7da8e57243cd7e824c263f // video
+    //https://api.jsonbin.io/b/5f7da6f065b18913fc5bfe74 // article
```

Comment:
Good idea to add examples in comments like this. It really helps when parsing the code.